app/policies/participant_policy.rb

```diff
@@ -27,7 +27,8 @@ def update?
 
   def user_can_access?(participant)
     return false unless participant.participant?
-    return false if participant.participant_profile.withdrawn?
+    # TODO: is this correct? assumes we're only looking at mentors and ECTs?
+    return false if participant.participant_profiles.ecf.first.withdrawn?
```

Comment:
I think it probably works. Although I wonder what happens if the induction coordinator tries to access index page with a single withdrawn user and a few active ones. Possibly something scary. 

Does all of this logic make sense? I think not quite - we wrote the policy before adding NPQ profiles, so it is not aware of them. I would say for time being we probably should return false if the user is npq, just to be safe.

Reply:
Think that would then prevent us from doing what this PR is intended to fix though, because then if a user was a NPQ we wouldn't be able to administer their mentor profile if that makes sense?

Reply:
Agree that I think this area needs more thought at some point

Reply:
Ah, true that. Perhaps what we want here is to check 'does this user have any active ect profiles? if not, then return false'?